config/constants.php

```diff
@@ -3,8 +3,8 @@
 
```

Comment:
Using constants for User Account status, we shall be including this in login and other places wherever it is required or in middleware.